companion/src/qcustomplot/GPL.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Was it a thirdparty? If yes it should be in its own directory (as I did for FatFS / Lua / CoOS / etc.), no?